src/hyperion/parameters/plan_specific/gridscan_internal_params.py

```diff
@@ -59,6 +60,12 @@ def _preprocess_experiment_params(
         cls,
         experiment_params: dict[str, Any],
     ):
+
+        #Force first grid scan to have even number of rows so next grid starts at same point
+        if experiment_params['y_steps']%2:
```

Comment:
Should: This has different whitespace than the one below. I feel like the one below is more correct

Reply:
Good idea, no point throwing an error for the regular FGS here.